libraries/adaptive-expressions/src/builtinFunctions/contains.ts

```diff
@@ -35,10 +35,15 @@ export class Contains extends ExpressionEvaluator {
         const { args, error: childrenError } = FunctionUtils.evaluateChildren(expression, state, options);
         let error = childrenError;
         if (!error) {
-            if ((typeof args[0] === 'string' && typeof args[1] === 'string') || Array.isArray(args[0])) {
+            if (typeof args[0] === 'string' && typeof args[1] === 'string') {
                 found = args[0].includes(args[1]);
-            } else if (args[0] instanceof Map) {
```

Comment:
Does it seem like this logic is getting lost in this refactor?

Reply:
Yes. The logic of `contains` of Map would be handlered in `accessProperty`